tools/rna_tools/ribotaper/tool_dependencies.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="samtools" version="0.1.19">
+        <repository name="package_samtools_0_1_19" owner="devteam" />
+    </package>
+    <package name="bedtools" version="2.19.1">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url" sha256sum="17253439e8191cb51a8bffba640c580e09ba158d3a0c66649e53d23ee3e50f77">
+                    https://depot.galaxyproject.org/software/bedtools/bedtools_2.19_src_all.tar.gz
+                </action>
+                <action type="shell_command">make</action>
+                <action type="move_directory_files">
+                    <source_directory>bin</source_directory>
+                    <destination_directory>$INSTALL_DIR/bin</destination_directory>
+                </action>
+                <action type="set_environment">
+                    <environment_variable name="BEDTOOLS_ROOT_DIR" action="set_to">$INSTALL_DIR</environment_variable>
+                </action>
+            </actions>
+        </install>
+        <readme>BEDTools is a collection of utilities for comparing, summarizing, and intersecting genomic features in BED, GTF/GFF, VCF and BAM formats.</readme>
+    </package>
+    <package name="R_3_0_1" version="3.0.1">
+        <repository name="package_r_3_0_1" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="R_dependencies" version="1.2">
+        <install version="1.0">
+            <actions>
+                <action type="setup_r_environment">
+                    <repository name="package_r_3_0_1" owner="iuc">
+                        <package name="R_3_0_1" version="3.0.1" />
+                    </repository>
+                    <package>https://cran.r-project.org/src/contrib/codetools_0.2-14.tar.gz</package>
```

Comment:
Could you put these package please into https://github.com/galaxyproject/cargo-port. This way the tarballs are sustainable.